src/service/resources-impl.js

```diff
@@ -620,9 +620,7 @@ export class Resources {
    * @param {number|undefined} newHeight
    * @param {number|undefined} newWidth
    * @return {!Promise}
-   * @protected
```

Comment:
It's because the code inserts an <amd-ad> element into the DOM and then wants to attempt to change its size. I couldn't find a way of getting a BaseElement/BaseCustomElement instance from an Element that has just been inserted. Maybe I missed something?

I could add an attemptChangeHeight-like function to BaseElement that allows an Element other than the one owned to be specified, but seems more straightforward to just make this function here public.